common/src/main/java/whocraft/tardis_refined/mixins/CubeMapMixin.java

```diff
@@ -0,0 +1,22 @@
+package whocraft.tardis_refined.mixins;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.math.Matrix4f;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.CubeMap;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(CubeMap.class)
+public class CubeMapMixin {
```

Comment:
Is this still needed? Doesn't seem to do anything.

Reply:
You are correct - this needs removed